astropy/units/tests/test_quantity_array_methods.py

```diff
@@ -553,8 +553,8 @@ def test_slice_to_quantity(self):
         """
         Regression test for https://github.com/astropy/astropy/issues/2003
         """
-
```

Comment:
@MridulS, what if you defined a `rng` fixture in the main `conftest.py`, then you can just add an `rng` argument to any pytest function to get the rng. Should save a lot of lines!